examples/api/src/app/api/open-graph/lib/url-handlers/index.ts

```diff
@@ -1,18 +1,20 @@
 import { UrlHandler } from "../../types/url-handler";
 import caip19 from "./caip-19";
-import fallback from "./fallback";
 import opensea from "./opensea";
 import zora from "./zora";
 import zoraPremint from "./zora-premint";
 import imageFileUrl from "./image-file";
+import metascraper from "./metascraper";
+import localFetch from "./local-fetch";
 
 const handlers: UrlHandler[] = [
   opensea,
   zoraPremint,
   zora,
   caip19,
   imageFileUrl,
-  fallback,
+  localFetch,
+  metascraper,
```

Comment:
@davidfurlong note the priority change here, let me know your thoughts about using microlink as a fallback after trying local fetching + parsing first

Reply:
I like it; curious why you chose this lib over the metascraper one - it'd be nice to have the same syntax

Reply:
I agree common syntax would be nice. The primary reason why I introduced the local fetch using a different library was because the microlink API, which is based on metascraper, was returning a lot of incomplete metadata for some popular sites (github, amazon, dune)

With that being said, I am curious to see if we get different results running metascraper ourselves so I will investigate further.